interface/reports/patient_list_creation.php

```diff
@@ -649,8 +673,9 @@ function printForm(){
                                     <td ><?php echo text($patDetailVal['patient_id']); ?></td>
                                     <td ><?php echo text($patDetailVal['patient_age']);?></td>
                                     <td ><?php echo text($patDetailVal['patient_sex']);?></td>
-                                    <td ><?php echo generate_display_field(array('data_type' => '36','list_id' => 'race'), $patDetailVal['patient_race']); ?></td>
-                                    <td colspan=5><?php echo text($patDetailVal['users_provider']);?></td>
+                                    <td ><?php echo text($patDetailVal['patient_race']);?></td>
+                                    <td ><?php echo text($patDetailVal['patient_ethnic']);?></td>
```

Comment:
race and ethnicity are multiselect (can choose more than one of them in patient demographics), which will make filtering tougher i think (maybe term with % on each side if it will suffice in query search). To display it, should run through generate_display_field with data_type of 33